src/sql/sqlops.d.ts

```diff
@@ -209,6 +209,31 @@ declare module 'sqlops' {
 		id: string;
 	}
 
+	/**
+ 	* Options for the actions that could happen after connecting is complete
+ 	*/
+	export interface IConnectionCompletionOptions {
+		/**
+		 * save the connection to MRU and settings (only save to setting if profile.saveProfile is set to true)
+	 	*/
+		saveTheConnection: boolean;
+
+		/**
+	 	* open the dashboard after connection is complete
+	 	*/
+		showDashboard: boolean;
+
+		/**
+		 * Open the connection dialog if connection fails
+		 */
+		showConnectionDialogOnError: boolean;
```

Comment:
I think this should default to true. Please document defaults for all these options. I wonder for this one - in the case of the `openConnectionDialog` I'd expect it has to be true, are we including it just because future APIs might want it to be configurable? 

We could consider just hiding this option for now and setting it to `true` in the mainThreadConnectionManagement class